mlibrary/pride
# Removing `Pride.Util.MultiSearch` and `Pride.Util.SearchSwitcher`

# Overview
`Pride.Util.MultiSearch` and `Pride.Util.SearchSwitcher` were only ever used in Search, and never contained any logic coming from Pride itself. The `MultiSearch` and `SearchSwitcher` classes [have been added to Search](https://github.com/mlibrary/search/pull/426), and can now remove these functions.

## Testing
1. Build the repository (`npm run build`).
2. Run the tests to make sure they pass (`npm run test`).
   * Break the new/updated unit tests to make sure they're working properly.
3. Navigate to your local [Search repository](https://github.com/mlibrary/search) and apply the newly generated files:
   1. Open `./package.json` and add `#pride-util-multisearch-searchswitcher` at the end of the `pride` dependency URL:
      ```bash
      "pride": "git+https://github.com/mlibrary/pride.git#pride-util-multisearch-searchswitcher"
      ``` 
   2. Do a clean install of Search:
      ```bash
      rm -rf node_modules && rm package-lock.json && npm install
      ``` 
   3. Start Search (`npm start`) and look around [the site](http://localhost:3000/everything). Check to see if there are any console errors, and everything still works as expected.
- Make sure the PR is consistent in these browsers:
